src/lib/components/colors/index.svelte

```diff
@@ -9,6 +10,7 @@
 <h2 class="sr-only">Check the contrast ratio between two colors</h2>
 
 <SupportWarning format={$format} />
+<ImplementationWarning format={$format} />
```

Comment:
May not be new to this PR, but we have a few layout issues when multiple warnings appear next to each other. (This is easier to test locally if you temporarily change `{#if !isSupported}` to `{#if true}` in `SupportWarning.svelte`.)

![Screenshot 2023-11-29 at 2 51 23 PM](https://github.com/oddbird/oddcontrast/assets/552316/29ffdf1c-f90e-4769-9539-ddc41ba519d3)

![Screenshot 2023-11-29 at 2 51 46 PM](https://github.com/oddbird/oddcontrast/assets/552316/9ae77b30-6e4b-4fa4-89f6-6991c90e790e)

Reply:
No, this isn't new to the PR. I'll leave it to whoever works on cleaning up the design on this to whether they want to fix it on this ticket or on a new one.

Reply:
Moved to #152